api/USB.json

```
@@ -40,6 +40,55 @@
           "deprecated": false
         }
       },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "tags": [
+            "web-features:webusb"
+          ],
+          "support": {
+            "chrome": [
+              {
+                "version_added": "118",
+                "partial_implementation": true,
+                "notes": "Available in dedicated workers and WebExtension service workers, not available in shared workers and normal service workers."
+              },
+              {
+                "version_added": "70",
+                "partial_implementation": true,
+                "notes": "Available in dedicated workers, not available in shared workers and service workers."
+              }
+            ],
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": {
+              "version_added": false,
+              "notes": "WebView exposes this interface, but does not support WebUSB. See <a href='https://crbug.com/933055'>bug 933055</a>."
```

Since that bug is tracking the implementation, I think we could use `impl_url`:

```suggestion
              "notes": "WebView exposes this interface, but does not support WebUSB. ",
              "impl_url": "https://crbug.com/933055"
```